chsdi/views/features.py

```diff
@@ -258,7 +258,7 @@ def _identify(request):
         layerId in layerIds
         if models_from_bodid(layerId) is not None
     ]
-    if models is None:
+    if models is None:  # pragma: no cover
```

Comment:
This is an error in the code as models will always be a list. We should test ```if len(models) == 0:``` instead.